src/stylesheets/base.scss

```diff
@@ -8,22 +8,22 @@
 @import url(https://www.thoughtworks.com/css/k8s_production/screen.css);
 @font-face {
   font-family: 'Oslo sans';
-  src: url('./fonts/OsloSans-Regular.woff2') format('woff2');
+  src: url('./images/OsloSans-Regular.woff2') format('woff2');
```

Comment:
Hvorfor `images` her? Ligger ingen slik katalog under `stylesheets`?

Reply:
Se commit-meldingen for siste commit. Jeg har ikke funnet beste løsning for fonts og ny Webpack. Jeg tar gjerne i mot forbedringsforslag. Fontene blir servert under `/images` i dagens versjon også.